### DISCUSSION TITLE
NFR - Performance Testing

### DISCUSSION THREAD

**Discussion by DanielaWuensch** [eclipse-edc/Connector#1669] [Ideas]
# Feature Request
Non-functional Requirement - Performance Testing
Representative performance test cases and performance test infrastructure must be in place and tests should be executed regularly

## Which Areas Would Be Affected?
all, including DPF, CI, build, transfer, etc._

## Why Is the Feature Desired?
Performance Requirement

## Solution Proposal
Representative performance test cases and performance test infrastructure 

## Type of Issue
non-functional requirement

## Checklist
Representative performance test cases and performance test infrastructure in place

prerequisite for Issue https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues/1637

**Comment by zeier** [comment]
For currently available test infrastructure, please refer to the decision record [Performance Testing with Gatling](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/tree/main/docs/developer/decision-records/2022-03-02-performance-tests)
